pom.xml

```diff
@@ -135,11 +135,11 @@
         </dependency>
 
         <!-- Logging -->
-        <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-log4j12</artifactId>
-            <version>${slf4j.version}</version>
-        </dependency>
+        <!--<dependency>-->
```

Comment:
Why is this commented?

Reply:
dependency is present in sprintboot by default